claim/gql_mutations.py

```diff
@@ -268,9 +270,19 @@ def create_attachment(claim_id, data):
     data["claim_id"] = claim_id
     from core import datetime
     now = datetime.datetime.now()
-    if ClaimConfig.claim_attachments_root_path:
-        # don't use data date as it may be updated by user afterwards!
-        data['url'] = create_file(now, claim_id, data.pop('document'))
+    general_type = data['general_type']
+    data['module'] = 'claim'
+    if general_type == GeneralClaimAttachmentType.URL:
+        parsed_url = urlparse(data['url'])
+        if (ClaimConfig.allowed_domains_attachments and
```

Comment:
If config there but empty, is this checked?

Reply:
ClaimConfig.allowed_domains_attachments defaults to false if list is empty, so whole expression is false.